### ISSUE TITLE
CompGO results

### ISSUE THREAD

**Issue by Ellior2** [RobertsLab/project-pacific.oyster-larvae#20]
I put the set of proteins detected in each of my 22 samples into CompGO and downloaded the report for biological processes. They are listed in this folder: https://github.com/RobertsLab/project-pacific.oyster-larvae/tree/master/DDA_2016/Compgo_results

@sr320 @emmats @kaitlynrm

**Comment by sr320** [comment]
For a single silo, provide a list of corresponding GO Slim Terms per day.

**Comment by emmats** [comment]
Can you figure out which ones are present at every stage and which are appearing and disappearing?

**Comment by sr320** [comment]
re @emmats comment - Do this first at GO Slim level.

**Comment by kaitlynrm** [comment]
I'm trying to ID which ones appeared (from 0 at day 1) right now! I should clarify that I'm looking at ones that appeared for only 1 day and could be considered "uniquely expressed".

**Comment by kaitlynrm** [comment]
I should clarify that I'm looking at ones that appeared for only 1 day and could be considered "uniquely expressed".

**Comment by Ellior2** [comment]
https://ellior2.github.io/CompGo_Silo2-23C/

**Comment by sr320** [comment]
Those are not slim terms?

**Comment by Ellior2** [comment]
Sorry not sure what you are talking about then...

**Comment by sr320** [comment]
You might recall from class 😄  that GO terms have parent terms and there are a set of ~15 top level terms that simplify the classification. 
There are a few ways to get corresponding GO slims terms..
ie 
https://www.animalgenome.org/bioinfo/tools/countgo/
https://www.ebi.ac.uk/QuickGO/GMultiTerm#tab=introduction

I would suggest SQLShare
You would just join your GO IDs with 
https://sqlshare.uw.edu/detail/sr320/GO_to_GOslim
to get GOSlim_bin (focusing only on aspect P).

**Comment by Ellior2** [comment]
https://ellior2.github.io/GoSlim_Silo2-23C/

**Comment by sr320** [comment]
Looks good
Can you make a chart where days are on the X axis?

**Comment by Ellior2** [comment]
Yep! https://raw.githubusercontent.com/RobertsLab/project-pacific.oyster-larvae/master/DDA_2016/GO_slim/Silo2-23CDayX-axis.JPG

**Comment by Ellior2** [comment]
I just went for it and did all of them. Now we can compare resource allocation between the three silos through time. https://ellior2.github.io/GoSlim/
